cmd/cartesi-rollups-cli/root/deps/deps.go

```diff
@@ -0,0 +1,20 @@
+// (c) Cartesi and individual authors (see AUTHORS)
+// SPDX-License-Identifier: Apache-2.0 (see LICENSE)
+
+package deps
+
+import (
+	"github.com/cartesi/rollups-node/cmd/cartesi-rollups-cli/root/deps/start"
+	"github.com/spf13/cobra"
+)
+
+var Cmd = &cobra.Command{
+	Use:   "deps",
```

Comment:
I'm not found of having two levels off commands. Why not just `cartesi-rollup-cli start-deps`?.

Reply:
I will change this. I started with "deps start" and "deps stop" , and then did not change after ditching the stop.